CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
+-  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- A user can also leverage [.dotflies](https://www.gitpod.io/docs/configure/user-settings/dotfiles) to share global settings for own account among all workspaces. A dotfile install script should copy settings from a dotfile repo to predefined locations as specified below. Dotfiles are guaranteed to apply before any JetBrains product startup.  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+-  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | Predefined location | JetBrains global settings for | ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | ------------- | ------------- | ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | `~/.gitpod/jetbrains/options`<br>Example: `/home/gitpod/.gitpod/jetbrains/options` | Each JetBrains product configured in a Docker image. |  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | `~/.gitpod/jetbrains/{product}/options`<br>Example: `/home/gitpod/.gitpod/jetbrains/intellij/options` | A specific JetBrains product configured in a Docker image. | ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | `/{project}/.gitpod/jetbrains/options`<br>Example: `/workspace/gitpod/.gitpod/jetbrains/options` | Each JetBrains product configured in a Git repo of your project. |  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | `/{project}/.gitpod/jetbrains/{product}/options`<br>Example: `/workspace/gitpod/.gitpod/jetbrains/intellij/options` | A specific product configured in a Git repo of the project. | ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+-  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- For an example see https://github.com/akosyakov/parcel-demo/tree/jb-proxy-test applying proxy settings via Docker image. ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- The processes in the workspace are given up to 3 minutes after receiving SIGTERM. ([#14498](https://github.com/gitpod-io/gitpod/pull/14498)) - [@utam0k](https://github.com/utam0k)
```

Comment:
@akosyakov, it seems the Release Notes don't accept multiple lines as in https://github.com/gitpod-io/gitpod/pull/14566.

Reply:
Cc @filiptronicek @loujaybee 

we can adjust PR, Lou will need to put it better next week anyway)

Reply:
Let's adjust it today, otherwise, @roboquat will create a new PR with this same content tomorrow.